src/utils/eventAlarmListener.ts

```diff
@@ -30,6 +33,10 @@ export default async function eventAlarmListener(alarm) {
       ...(newIndex === 0 && { project_id: 0 })
     })
     await storage.set("arebyte-audience-session", updatedSession)
+    await storage.set(
```

Comment:
Think we might need to refactor this in light of what we were working on this morning. Think this should all move the db

Reply:
done :)